pkg/ocmagenthandler/ocmagenthandler_deployment.go

```diff
@@ -206,14 +205,18 @@ func (o *ocmAgentHandler) ensureDeployment(ocmAgent ocmagentv1alpha1.OcmAgent) e
 	populationFunc := func() appsv1.Deployment {
 		return buildOCMAgentDeployment(ocmAgent)
 	}
+
+	envs := o.buildEnvVars()
```

Comment:
as `buildEnvVars` returns `[]corev1.EnvVar`, should this be renamed to `envVars`?